Sprint-2/implement/to-pounds.js

```diff
@@ -4,3 +4,26 @@
 // You will need to declare a function called toPounds with an appropriately named parameter.
 
 // You should call this function a number of times to check it works for different inputs
+
+function toPounds(pounds){
+
+    pounds = pounds.substring(
+        0,
+        pounds.length - 1
+      );
+      
+      const paddedPenceNumberString = pounds.padStart(3, "0");
+      pounds = paddedPenceNumberString.substring(
+        0,
+        paddedPenceNumberString.length - 2
+      );
+      
+      const pence = paddedPenceNumberString
+        .substring(paddedPenceNumberString.length - 2)
+        .padEnd(2, "0");
```

Comment:
Do we really need `.padEnd(2, "0")`?

Reply:
We don't need padEnd() necessarily because "pence" will always have two digits. So I removed padEnd().